pushsync/pusher_test.go

```diff
@@ -226,29 +241,11 @@ func TestPusher(t *testing.T) {
 			return nil
 		}
 		// check outgoing chunk messages
-		cur := int(binary.BigEndian.Uint64(chmsg.Addr[:8]))
-		if cur > max {
-			errf("incorrect order of chunks from db chunk #%d before #%d", cur, max)
-			return nil
-		}
-		v, found := sent.Load(cur)
-		previouslySentAt, repeated := sendTimes[cur]
-		if !found {
-			if !repeated {
-				errf("chunk #%d not sent but received", cur)
-			}
+		idx := int(binary.BigEndian.Uint64(chmsg.Addr[:8]))
+		if idx > max {
+			errf("incorrect order of chunks from db chunk #%d before #%d", idx, max)
```

Comment:
Why can all of this simply be removed?

Reply:
so shall i keep the too early to retry   logic even if we do dynamic retry intervals? maybe you are right. Is that what  you mean?

Reply:
I didn't go into it. I'm just asking because there's a block of code gone without any mapping to why. As mentioned, there's too many ends to tie together when we PR like this.